src/views/mfa-verify/PushForm.js

```diff
@@ -146,6 +139,26 @@ function (Okta, CookieUtil, Util, NumberChallengeView) {
     },
     clearWarnings: function () {
       this.$('.okta-form-infobox-warning').remove();
+    },
+    handleRejectStateChange: function (state, isMfaRejected) {
+      if (isMfaRejected) {
+        this.setSubmitState(isMfaRejected);
+        this.setRejectedErrorMessage();
+      }
+    },
+    setRejectedErrorMessage: function () {
+      // If rejection is due to outdated app, show error message per platform
+      // else show user rejected message.
+      if (this.options.appState.get('lastAuthResponse').factorResultMessage
```

Comment:
Not making this a derived prop since this is not used in many places.

Reply:
nit: I would still vote to add the derived property so if we need it in the future it is already there. Otherwise if someone else adds this usage might not get replaced...
This is a nit, so no prob at all if we keep it as is